src/components/Dialog.vue

```diff
@@ -29,6 +29,7 @@
     <div class="dialog__container">
       <h2 class="dialog__title">{{ titleText }}</h2>
       <p>{{ subtitleText }}</p>
+	  <input type="hidden" name="user" :value="userId"/>
```

Comment:
For some password managers and from w3c recommendations, it also should has `autocomplete` attribute.

```suggestion
	  <input type="hidden" name="user" autocomplete="username" :value="userId"/>
```